pkgs/development/libraries/vrb/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "vrb-${version}";
+  version = "0.5.1";
+
+  src = fetchurl {
+    url = "http://vrb.sourceforge.net/download/${name}.tar.bz2";
```

Comment:
Please use `mirror://sourceforge`.

Reply:
VRB doesn't host files under https://sourceforge.net/projects/vrb/files/, but only through its sourceforge-hosted website (http://vrb.sourceforge.net/). I couldn't find a way to get these files using a mirror like http://downloads.sourceforge.net/, so I presume it is not possible to use `mirror://sourceforge`.

Reply:
You are right, thank you.